pvlib/iotools/pvgis.py

```diff
@@ -424,6 +441,12 @@ def get_pvgis_tmy(latitude, longitude, outputformat='json', usehorizon=True,
         base url of PVGIS API, append ``tmy`` to get TMY endpoint
     timeout : int, default 30
         time in seconds to wait for server response before timeout
+    utc_offset: int, default 0
+        Use to specify a timezone other than the default UTC zero. Will force
+        year to ``coerce_year`` if not zero.
```

Comment:
This parameter names and description don't make any mention of rolling the data.  I think at least the parameter description should explain the rolling.  Ideally the parameter name would somehow reflect it as well, although I'm having trouble coming up with a good name.  Somehow `roll_according_to_utc_offset` doesn't quite roll off the tongue...

Reply:
I was going to leave good enough alone, but since we have more suggestions: I would roll automatically based on longitude.

Reply:
I can add explanation of rolling and maybe somewhere we should add description that TMY is always expected to start and end at midnight on Jan 1/December 31?

Reply:
>roll automatically based on longitude.

I don’t want to make rolling mandatory. Using longitude is clever, but not all time zones are strictly longitudinal, and I’ve had issues using that approach.

Reply:
No problem.

Reply:
> Somehow `roll_according_to_utc_offset` doesn't quite roll off the tongue...

How about `align_to_local_noon`?

Reply:
@adriesse  I changed it to `roll_utc_offset` is that okay? It's both a verb `roll` and a number `utc_offset`. I like your suggestion, and thought my tests should probably have check transit times, which is the real test of whether it's in the right time zone, but that was a bridge too far.

Reply:
It's all good.  No strong feelings about any of this.